### ISSUE TITLE
 No swift syntax highlighting support in Sublime Merge

### ISSUE THREAD

**Issue by bgeron** [sublimehq/sublime_merge#1503]
Hiya,

I'm reopening #724 to ask for Swift highlighting support :) It's the #12 language on TIOBE, and the main language for iOS app development. It doesn't have to be perfect, but I think it's a bit overdue for an all-round good Git UI.

The lack of Swift support is making me rethink using Sublime Merge at work — even though I'm really happy with it otherwise and I use SMerge every day, both professionally and privately.

Cheers, Bram

P.S. Thank you for making such a great product <3

**Comment by TheSecEng** [comment]
Do you have Sublime Text installed with Swift syntax ?

I believe merge can pick up that syntax.

**Comment by bgeron** [comment]
No, I don't. Ideally Swift would be included by default like the other mainstream languages. I don't have a license for Text;  ideally I can avoid it.

**Comment by TerminalFi** [comment]
You don't need a license FYI. You could theoretically installed sublime text and install swift, then merge with us it. You can then uninstall Sublime Text and I believe it continues to work.
